services/slam/builtin/builtin.go

```diff
@@ -686,6 +694,26 @@ func (slamSvc *builtIn) StopSLAMProcess() error {
 	return nil
 }
 
+func (slamSvc *builtIn) getLazyPngImage(ctx context.Context, cam camera.Camera) ([]byte, error) {
+	// We will hint that we want a PNG.
+	// The Camera service server implementation in RDK respects this; others may not.
+	img, release, err := camera.ReadImage(
+		gostream.WithMIMETypeHint(ctx, utils.WithLazyMIMEType(utils.MimeTypePNG)), cam)
+	if err != nil {
+		return nil, err
+	}
+	defer release()
```

Comment:
I'm concerned about calling `release()` before we've written out the image to a file. Maybe `getLazyPngImage()` should return `release` instead.

Reply:
Do you want the `getSimultaneousColorAndDepth` to also return `release` and call `defer release()` after the call of 

```go
images, err := slamSvc.getSimultaneousColorAndDepth(ctx, cams)
```
?

Reply:
Good point, I wonder if that is a bug currently! I'm honestly not sure what the release function does, and I couldn't figure it out by reading the code. Maybe we could ask Bijan here? Or ask Eric next week.

Reply:
Based on our discussion (i.e. rumors that `release` could be ignored and doesn't need to be called): Removing all calls of `release` for now. Will ask Eric next week